vj4/ui/templates/contest_detail.html

```diff
@@ -8,19 +8,19 @@
     <div class="section">
       <div class="section__body typo">
         {{ tdoc['content']|markdown }}
-      {% if handler.is_live(tdoc) %}
+      {% if handler.is_ongoing(tdoc) %}
```

Comment:
???

Reply:
"Live" is not a correct usage. Rename to "Ongoing".